app/javascript/elm/src/Main.elm

```diff
@@ -66,6 +67,7 @@ defaultModel =
     { page = Mobile
     , key = Nothing
     , sessions = []
+    , availableSessionsCount = 0
```

Comment:
If I was reading this out of context "available" would be confusing. I'd suggest something like "fetchable" unless you have a better name. Maybe @gitjul has a suggestion?

Reply:
I was thinking of going simpler and using the word `total`, but `fetchable` is the most obvious and explicit so I'd just go with that.